src/InvokePlaster.ps1

```diff
@@ -1512,3 +1514,14 @@ function GetPlasterManifestPathForCulture([string]$TemplatePath, [ValidateNotNul
 
     $null
 }
+
+
+# Dot source the individual module command scripts.
+. .\NewPlasterManifest.ps1
```

Comment:
We don't dot source these here but in the psm1 file.

Reply:
I don't understand this one... thanks a lot for cleaning it.